extensions/amp-date-countdown/validator-amp-date-countdown.protoascii

```diff
@@ -37,6 +37,7 @@ tags: {  # <amp-date-countdown>
     value_casei: "seconds"
   }
   attrs: {  # attribute is not used in version 0.1 of the component (see #33480)
+    disabled_by: "amp4email"
```

Comment:
This component is not enabled in amp4email so this is not needed.

Reply:
Thank you.  I have removed in the latest revision.